### PR TITLE
Keeping the relay that responded to syncCheck and syncCall - until forging is done.

### PR DESCRIPTION
**Pull request by kristjank**
## Proposed changes
Using the relay that responded to synCheck and making sure the relay is synced. Also no host choosing between getting network state and transaction from pool. Saving time and sticking with the host that responded and is synced.

## Types of changes
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (improve a current implementation without adding a new feature or fixing a bug)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build (changes that affect the build system)
- [ ] Docs (documentation only changes)
- [ ] Test (adding missing tests or fixing existing tests)
- [ ] Other... Please describe:

## Checklist
- [x] I have read the [CONTRIBUTING](https://docs.ark.io/developers/guidelines/contributing.html) documentation
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by kristjank** [comment]
@faustbrian

File: packages/core-forger/__tests__/client.test.js [hunk 1]
```diff
@@ -79,6 +85,10 @@ describe('Client', () => {
     })
 
     describe('when the host is available', () => {
+      beforeEach(async () => {
```
Comment (faustbrian) [inline packages/core-forger/__tests__/client.test.js]: There is only 1 test in this `describe` block, `await client.__chooseHost()` should be called inside the `it` block.

File: packages/core-forger/__tests__/client.test.js [hunk 2]
```diff
@@ -98,6 +108,10 @@ describe('Client', () => {
...
```
Comment (faustbrian) [inline packages/core-forger/__tests__/client.test.js]: There is only 1 test in this `describe` block, `await client.__chooseHost()` should be called inside the `it` block.

File: packages/core-forger/__tests__/client.test.js [hunk 3]
```diff
@@ -45,7 +45,13 @@ describe('Client', () => {
...
```
Comment (faustbrian) [inline packages/core-forger/__tests__/client.test.js]: There is only 1 test in this `describe` block, `await client.__chooseHost()` should be called inside the `it` block.

**Comment by Jarunik** [comment]
I will try set it up again on new devnet and test there. Already killed nodes on current devnet.

**Pull request merged**